inc/sub-headerException.php

```diff
@@ -0,0 +1,124 @@
+<div class="newsSubHeader clearfix">
+<div class="innerPadding clearfix">
+<div class="title-page no-padding-left col-xs-12  col-sm-4 col-md-5 col-lg-5">
+	
+    <?php if( is_single($post)){ ?>
+    	<h2 class="name-site"><a href="/news/">News &amp; events</a></h2>
+      <?php }else{ ?>
+    
+    <h1 class="name-site"><a href="/news/">News &amp; events</a>
+	<?php 
+	if(is_category()){
+	 printf('<span>'. ': ' . single_cat_title( '', false ) . '</span>' ); 
+	}  ?>        
+    </h1>
+	
+   <?php } ?>
+    
+ </div>   
+<div class="socialNav singleSocialNav hidden-xs not_on_phone socialNav col-xs-12 col-sm-8 col-md-7  col-lg-7 clearfix ">
```

Comment:
In addition to avoiding camel case, can you change the underscores (unless it existed and would require more work)? ideally all css classes would be dashed.